majority-multisign/test/Spec/Direct.hs

```diff
@@ -190,7 +190,14 @@ tests =
     validatorScript = getValidator (validator initialParams)
 
 arbitraryDatumFrom :: [PubKeyHash] -> Gen Schema.MajorityMultiSignDatum
-arbitraryDatumFrom sigs = Schema.MajorityMultiSignDatum <$> sublistOf sigs
+arbitraryDatumFrom sigs =
+  Schema.MajorityMultiSignDatum . takeNatural Schema.maximumSigners
+    <$> sublistOf sigs
+
+takeNatural :: Natural -> [a] -> [a]
+takeNatural [nat| 0 |] _ = []
```

Comment:
Can use [`zero`](https://staging.plutus.iohkdev.io/doc/haddock/plutus-tx/html/PlutusTx-Numeric.html#t:AdditiveMonoid) over quazi here.

Reply:
Good point, I'll do that.